CMakeLists.txt

```diff
@@ -230,6 +231,58 @@ endif()
 if (APPLE)
     target_link_libraries(${PROJECT} "-framework Cocoa")
 
+    file(GLOB OLOCO_RESOURCES "${CMAKE_CURRENT_SOURCE_DIR}/data/language/*")
+
+    set(BUNDLE_RESOURCE "${PROJECT}.app/Contents/Resources")
+    set(SOURCE_DATA_DIR "${CMAKE_CURRENT_SOURCE_DIR}/data")
+    set(SOURCE_ICON_DIR "${CMAKE_CURRENT_SOURCE_DIR}/resources/logo")
+
+    file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/distribution/macos/AppIcon.iconset DESTINATION ${CMAKE_BINARY_DIR})
+    set(ICON_TARGET "${CMAKE_BINARY_DIR}/AppIcon.iconset")
+    set(ICON_OUTPUT "${CMAKE_BINARY_DIR}/AppIcon.icns")
+
+    add_custom_command(OUTPUT ${ICON_OUTPUT}
+        COMMAND cp icon_x16.png   ${ICON_TARGET}/icon_16x16.png
+        COMMAND cp icon_x32.png   ${ICON_TARGET}/icon_16x16@2x.png
+        COMMAND cp icon_x32.png   ${ICON_TARGET}/icon_32x32.png
+        COMMAND cp icon_x64.png   ${ICON_TARGET}/icon_32x32@2x.png
+        COMMAND cp icon_x128.png  ${ICON_TARGET}/icon_128x128.png
+        COMMAND cp icon_x256.png  ${ICON_TARGET}/icon_128x128@2x.png
+        COMMAND cp icon_x256.png  ${ICON_TARGET}/icon_256x256.png
+        COMMAND cp icon_x512.png  ${ICON_TARGET}/icon_256x256@2x.png
+        COMMAND cp icon_x512.png  ${ICON_TARGET}/icon_512x512.png
+        COMMAND cp icon_x1024.png ${ICON_TARGET}/icon_512x512@2x.png
+        COMMAND iconutil -c icns ${ICON_TARGET}
+        WORKING_DIRECTORY ${SOURCE_ICON_DIR})
+
+    set(MACOSX_BUNDLE_COPYRIGHT "OpenLoco is licensed under the MIT License")
```

Comment:
Maybe move the static stuff straight into the template?

Reply:
Personally, I rather like that the template is reusable in its current form. Perhaps we should order these properties alphabetically, though.

Reply:
That's what Xcode seems to be doing as well: 
![image](https://user-images.githubusercontent.com/737603/53361952-5005fe80-3939-11e9-8d12-4bb6743560af.png)